blocks/procedures.ts

```diff
@@ -447,21 +465,21 @@ blocks['procedures_defnoreturn'] = {
   ...PROCEDURE_DEF_COMMON,
   /**
    * Block for defining a procedure with no return value.
-   * @this {Block}
    */
-  init: function() {
+  init: function(this: ProcedureBlock) {
```

Comment:
Can you make this one `ProcedureBlock&BlockSvg` to remove the cast below?

On the other hand, maybe that's more innaccurate and the problem is actually with types elsewhere (`setMutator`?).

Reply:
That would be more consistent with how I had handled earlier cases.  (Ahh, the challenges of trying to finish a task one started more than a month earlier…)

Done—here and for `blocks['procedures_defreturn']` below.